cmd/minishift/cmd/start_preflight.go

```diff
@@ -584,9 +587,8 @@ func checkVMDriver() bool {
 }
 
 func validateOpenshiftVersion() bool {
```

Comment:
maybe provide the version as a parameter, as it is used here as a non-reussable static function. This means we call `getOpenShiftVersion` again. If we provide this as a parameter we put that responsibility somewhere else.